samples/SkiaMultiplatformSample/src/wasmJsMain/kotlin/org/jetbrains/skiko/sample/Utils.kt

```diff
@@ -0,0 +1,56 @@
+package org.jetbrains.skiko.sample
+
+import org.khronos.webgl.ArrayBuffer
+import org.khronos.webgl.Int8Array
+import org.w3c.xhr.XMLHttpRequest
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+import kotlin.coroutines.suspendCoroutine
+import kotlin.wasm.unsafe.UnsafeWasmMemoryApi
+import kotlin.wasm.unsafe.withScopedMemoryAllocator
+
+suspend fun loadRes(url: String): ArrayBuffer {
+    return suspendCoroutine { continuation ->
+        val req = XMLHttpRequest()
```

Comment:
nothing blocking, just out of pure interest - why not fetch?  Worser external declarations in js/wasm?

Reply:
copy pasted from another place without thinking. Will fix :)

Reply:
fixed